README.md

```diff
@@ -81,9 +81,16 @@ issues or [on Discord][discord].
 ## Maintainers
 
 Open Source Pledge is not involved in any flow of funds and so we do not
-directly onboard maintainers. We encourage you to onboard to the following
-platforms to receive funding from Open Source Pledge Members:
+directly onboard maintainers. Please confirm with each open source project
+about their best means of recieving funds, as platforms and fiscal hosts have
+varying fee structures, and the recipient projects may best ways to maximize
```

Comment:
I think there's some kind of grammatical error here: “the recipient projects may best ways to maximize the impact of your contributions”.

Reply:
Fixed